microservice-schedule/src/test/resources/domain.xml

```diff
@@ -0,0 +1,261 @@
+<domain application-root="${com.sun.aas.instanceRoot}/applications" version="215" log-root="${com.sun.aas.instanceRoot}/logs">
```

Comment:
this could be probably much shorter. but I don't know what's essential for the Aquillian container.